### ISSUE TITLE
handler for resizing does not appeared!

### ISSUE THREAD

**Issue by vahiddabbagh** [mauricius/vue-draggable-resizable#234]
It seems that classes related to handle does not exist in my application

**Comment by TitanFighter** [comment]
Css added?

**Comment by mauricius** [comment]
Yes, it sounds like you're not adding the CSS file. See [Install and Basic Usage](https://github.com/mauricius/vue-draggable-resizable#install-and-basic-usage)
